projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-post-v1-1-endpoint.php

```diff
@@ -837,7 +837,10 @@ public function write_post( $path, $blog_id, $post_id ) {
 
 				$meta = (object) $meta;
 
-				if ( Jetpack_SEO_Posts::DESCRIPTION_META_KEY === $meta->key && ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+				if (
+					( Jetpack_SEO_Posts::DESCRIPTION_META_KEY === $meta->key || Jetpack_SEO_Posts::HTML_TITLE_META_KEY === $meta->key ) &&
```

Comment:
@robertsreberski - would it be worth updating the rest of these checks to use the `in_array` check like we did in one place so far?

Reply:
It feels like swapping to `in_array()` may work well especially if any other seo options are added subsequently -- at two I'm kinda a wash either way.

Just make sure we're setting the strict comparison third argument of `in_array()` to true.

https://www.php.net/manual/en/function.in-array.php

Reply:
#27409 plans to add another meta key, so the checks have been updated to use `in_array()` in c3b5bba

What about these two places?

https://github.com/Automattic/jetpack/blob/bc051609cc7972dc676ebbd0340ff7424bf40390/projects/packages/sync/src/class-defaults.php#L728

https://github.com/Automattic/jetpack/blob/b313bd253afdf919adf7a71a866d0ab85b3a7bda/projects/plugins/jetpack/sal/class.json-api-post-base.php#L297